app/src/modules/collections/routes/item.vue

```diff
@@ -151,7 +151,7 @@
 			v-model="edits"
 			:disabled="isNew ? false : updateAllowed === false"
 			:loading="loading"
-			:initial-values="item"
+			:initial-values="item || defaultItem"
```

Comment:
We shouldn't change the initial values object, as that would cause some issues in the staging of values and "was edited" tracking. v-form should already have access to all the default values, so we should handle this conditional problem in there (as a matter of fact, v-form is the component running the conditions, so that'd be the better place to tackle this regardless 🙂 )

Reply:
that makes total sense. I moved the logic to v-form, particularly the applyConditions function. What I did might not be the best approach tho 😅